app/views/coronavirus_form/testing_equipment.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :title do %><%= t('coronavirus_form.testing_equipment.title') %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('coronavirus_form.testing_equipment.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= render "govuk_publishing_components/components/title", {
+  title: t('coronavirus_form.testing_equipment.title'),
+  margin_top: 0
+} %>
+
+<p class="govuk-body govuk-!-margin-bottom-8">
+  <a class="govuk-link" href="https://www.gov.uk/guidance/help-the-government-increase-coronavirus-covid-19-testing-capacity" target="_blank">
+    <%= t('coronavirus_form.testing_equipment.link.label') %>
+  </a>
+</p>
+
+<%= render "govuk_publishing_components/components/button", {
+  text: t('coronavirus_form.testing_equipment.button.label'),
+  href: "/additional-product",
+  margin_top: 20,
```

Comment:
This value is out of range (since 9 is maximum) – I suspect we don't need it now that we use the `govuk-!-margin-bottom-8` override class on the element above.